cmd/acr/purge.go

```diff
@@ -614,6 +621,53 @@ func dryRunPurge(ctx context.Context, acrClient api.AcrCLIClientInterface, login
 	return deletedTagsCount, deletedManifestsCount, nil
 }
 
+func filterManifests(manifest acr.ManifestAttributesBase, cm customManifest, doNotDelete Set[string], candidatesToDelete []acr.ManifestAttributesBase) ([]acr.ManifestAttributesBase, Set[string], error) {
+	switch *manifest.MediaType {
+	case MediaTypeManifestList:
+		if manifest.Tags == nil {
```

Comment:
Is it possible that `manifest.Tags` is not nil but empty?

Reply:
discussed offline. will only check nil case.